src/index.ts

```diff
@@ -14,6 +14,11 @@ import {
 import { copy } from 'copy-anything';
 
 export default class SuperJSON {
+  /**
+   * If true, SuperJSON will make sure only one instance of referentially equal objects are serialized and the rest are replaced with `null`.
+   */
+  public dedupe = false;
```

Comment:
My gut instinct was to put this as an optional parameter to `serialize`, and call it something like `pruneReferentialEqualities`, in reference to `meta.referentialEqualities`. I'd also be fine with `dedupeReferentialEqualities`. It's a lot longer of a name, but more descriptive I guess? But also harder to parse, so i'm torn between the two. Any strong feelings from your end on this?

Reply:
Adding an optional argument to serialize could be seen as a breaking change in cases like `items.map(superjson.serialize.bind(superjson)` 🙃

Reply:
I see where you're coming from, but I don't necessarily agree with that strict of a definition of "breaking change" 😅